content/docs/machine-learning/creating-a-new-kubeflow-notebook.md

```diff
@@ -0,0 +1,103 @@
+---
+sidebar_position: 4
+title: Creating a new Kubeflow notebook
+description: Creating a new notebook in Civo's machine learning Kubeflow service
+---
+
+<head>
+  <title>Creating a new Kubeflow notebook | Civo Documentation</title>
+</head>
+
+To create a new Kubeflow notebook, you will need to provide a name, select a development environment, and make a few architectural decisions. All of these are explained below.
+
+## Notebook options
+
+:::tipNamespaces
+Always check on the top left corner of Kubeflow Dashboard if you are creating resources in the correct namespace.
```

Comment:
grammar: Should be "Always check on the top left corner of the KubeFlow Dashboard that you are creating resources in the correct namespace."